openquake/hazardlib/lt.py

```diff
@@ -921,7 +923,14 @@ def build(*bslists):
         branches = []
         for brid, value, weight in brlists:
             branches.append(Branch(bsid, brid, weight, value))
-        bset = BranchSet(utype, i, dict(applyToBranches=applyto))
+        # Check if the input for applyTo it is just a string (in this case, we
+        # assume it's the ID of a source) or a dictionary defining a more
+        # complex set of rules
+        if re.search('=', applyto):
+            tmp = toml.loads(applyto)
+            bset = BranchSet(utype, i, tmp)
+        else:
+            bset = BranchSet(utype, i, dict(applyToBranches=applyto))
```

Comment:
No, this is useless, applyToBranches is already implemented. I will fix the implementation and the tests,